tslint.json

```diff
@@ -6,7 +6,8 @@
     "jsRules": {},
     "rules": {
         "interface-name": [true, "never-prefix"],
-        "variable-name": [true, "allow-leading-underscore", "allow-pascal-case"]
+        "variable-name": [true, "allow-leading-underscore", "allow-pascal-case"],
+        "no-shadowed-variable": false
```

Comment:
Where was this one causing problems?

Reply:
in src/pars.ts where we have an argument `pars` but also a `pars` object at the module level. I (think I) saw you'd disabled this in wodin so figured I would too as it seems a bit of a nuisance